test/e2e/annotations/proxyssl.go

```diff
@@ -220,6 +223,6 @@ func assertProxySSLName(f *framework.Framework, host, sslName, ciphers, protocol
 				strings.Contains(server, fmt.Sprintf("proxy_ssl_ciphers %s;", ciphers)) &&
 				strings.Contains(server, fmt.Sprintf("proxy_ssl_protocols %s;", protocols)) &&
 				strings.Contains(server, fmt.Sprintf("proxy_ssl_verify %s;", verify)) &&
-				strings.Contains(server, fmt.Sprintf("proxy_ssl_verify_depth %d;", depth))
+				strings.Contains(server, fmt.Sprintf("proxy_ssl_server_name %s;", proxySSLServerName))
```

Comment:
Looking at this it seems you intended to add the line rather than overwriting it ?

Reply:
My bad. Will correct this.